src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -42,6 +42,7 @@ class Filter extends Component<Props> {
     return {
       hasForSaleArtworks: artist.counts.for_sale_artworks > 0,
       hasBuyNowArtworks: artist.counts.ecommerce_artworks > 0,
+      hasMakeOfferArtworks: artist.counts.has_make_offer_artworks,
```

Comment:
I'm unclear about this line. All the other `hasBLAHArtworks` keys in this object are being computed by comparing a count against zero - how is it that we have `artist.counts.has_make_offer_artworks` here as a boolean?

Reply:
I chose to use a boolean because the count in all the other validations don't accurately compare to the actual scopes for that purchase type (e.g. the ecommerce count is checking only if the artworks have ecommerce, not inventory, for sale, etc). These count fields are only validating whether to gray out the input so it's not a big deal either way. But in the future we wouldn't want anyone to use them as actual counts for the artworks, so it's better to use a boolean.